extensions/mssql/package.nls.json

```diff
@@ -82,6 +82,7 @@
 	"mssql.query.ansiNulls": "Enable SET ANSI_NULLS",
 	"mssql.query.alwaysEncryptedParameterization": "Enable Parameterization for Always Encrypted",
 	"mssql.ignorePlatformWarning": "[Optional] Do not show unsupported platform warnings",
+	"mssql.useNativeCredentials": "Migrate all passwords to native password storage",
```

Comment:
This isn't really an accurate description - the migration is just a one-time thing. Something like `Use platform native credential storage for storing passwords`.

Although that still is confusing since we use platform native storage for windows and MacOS regardless....